### PR TITLE
Glt 457 identity lists

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java [hunk 1]
```diff
@@ -352,13 +355,29 @@ public boolean aliasExists(String alias) throws IOException {
...
 
+  private static final String[] searchProperties = new String[] { "alias", "identificationBarcode", "name" };
+
+  /**
+   * Create a Hibernate criterion to search for all the properties our users want to search.
+   * 
+   * @param querystr
+   * @return
+   */
+  private Criterion searchRestructions(String querystr) {
```
Comment (callunity) [inline sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java]: "restructions" = portmanteau of "restrictions" and "structure"?

**Comment by callunity** [comment]
:+1:

File: sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java (hunk 1 above)
Comment (apmasell) [inline sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java]: I need spell check for variable names. Why isn't that a thing. :-/

**Comment by tonydebat** [comment]
👍

**Pull request merged**